spyderlib/widgets/arraybuilder.py

```diff
@@ -202,11 +206,11 @@ def __init__(self, parent, inline=True, offset=0):
         if inline:
             self._button_help.setToolTip(self._help_inline)
             self._text = NumpyArrayInline(self)
-            self._widget = self._text
+            self.widget = self._text

```

Comment:
To me the strangest thing is that the widget is not private while other components are. It looks like it's used somewhere else.
But I don't want to hurt your mixed feelings! :wink: You wrote this widget so you can decide.